File: app/src/main/java/ashunevich/uniconverter20/ui/AppViewModel.java
package ashunevich.uniconverter20.ui;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class AppViewModel extends ViewModel {
    private final MutableLiveData<Integer> selected = new MutableLiveData<>();
    private final MutableLiveData<String> postNumber = new MutableLiveData<>();

    public void setPostNumber(String textNumber) {
        postNumber.setValue(textNumber);
    }

    public LiveData<String> getPostedNumber() {
        return postNumber;
    }

    public void select(Integer pos) {
        selected.setValue(pos);
    }

    public LiveData<Integer> getSelected() {
        return selected;
    }

}

File: app/src/main/java/ashunevich/uniconverterKT/ui/AppViewModel.kt
package ashunevich.uniconverterKT.ui

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.LiveData

class AppViewModel : ViewModel() {
    private val selected = MutableLiveData<Int>()
    private val postNumber = MutableLiveData<String?>()
    fun setPostNumber(textNumber: String?) {
        postNumber.value = textNumber
    }

    val postedNumber: LiveData<String?>
        get() = postNumber

    fun select(pos: Int) {
        selected.value = pos
    }

    fun getSelected(): LiveData<Int> {
        return selected
    }
}